horovod/mxnet/mpi_ops.h

```diff
@@ -33,16 +33,18 @@ using namespace horovod::common;
 typedef ::mxnet::NDArray NDArray;
 typedef ::mxnet::Engine::CallbackOnComplete CallbackOnComplete;
 typedef Request::RequestType OperationType;
+typedef std::shared_ptr<MXTensor<NDArray>> MXTensor_sh_ptr;
 
 struct MpiOpsParam {
   NDArray* input;
   NDArray* output;
-  NDArray* cpu_tensor;
+  MXTensor_sh_ptr cpu_tensor;
```

Comment:
The name MXTensor_sh_ptr looks weired. Can you rename it, e.g. MXTensorSharedPtr?

Reply:
Sure. Thanks for the input.

Reply:
addressed in [ca9400a](https://github.com/horovod/horovod/pull/1304/commits/ca9400a9587cb0f8b2efe22f4a7016de06b74be2).